integration/container/restart_test.go

```diff
@@ -76,11 +76,10 @@ func TestDaemonRestartKillContainers(t *testing.T) {
 					d.Stop(t)
 				},
 			} {
+				c := tc
+				liveRestoreEnabled := liveRestoreEnabled
+				stopDaemon := stopDaemon
 				t.Run(fmt.Sprintf("live-restore=%v/%s/%s", liveRestoreEnabled, tc.desc, fnName), func(t *testing.T) {
```

Comment:
Should `tc.desc` here also be `c.desc`? (or `tc` if we decide to change `c := tc ->  tc := tc`

Reply:
It's `tc` now.